src/server/api/controllers/users.controller.js

```diff
@@ -6,35 +6,50 @@ const createUser = async (body) => {
   await knex('users').insert({
     full_name: body.full_name,
     email: body.email,
+    mobile: body.mobile,
     address: body.address,
     zipcode: body.zipcode,
     city: body.city,
     country: body.country,
     created_at: moment(Date.now()).format(),
   });
-
   return {
     successful: true,
   };
 };
-
-// eslint-disable-next-line
+const getUserById = async (user_id) => {
+  const id = +user_id;
```

Comment:
I'd suggest using regular expressions if parseInt isn't enough. 
https://stackoverflow.com/a/19715367

Don't rely on hacks like `+` in your code. They are unreliable and hard to understand.